src/goo/geometrypack/text/TextMeshGenerator.js

```diff
@@ -402,11 +402,11 @@ function meshesForText(text, font, options) {
 			meshBuilder.addMeshData(meshData, transform);
 		}
 	
-		if (options.backface)
-		  frontFace();
-      		else // If the back face shouldn't be visible, set extrusion to 0s
+		if (options.backface) {
+			frontFace();
```

Comment:
Thanks for the review, added the options.backface at the top, but when i change that the backface() is to be drawn when option.backface is true, I get the text backwards. The text isn't visible on the other side of the material due to the other face not being drawn.

With frontface drawn, no backface:
![skarmavbild 2016-07-13 kl 10 26 32](https://cloud.githubusercontent.com/assets/6060731/16796708/58649322-48e4-11e6-8a21-321e56fbc5df.png)

With backface drawn, no frontface:
![skarmavbild 2016-07-13 kl 10 25 50](https://cloud.githubusercontent.com/assets/6060731/16796776/9f20a148-48e4-11e6-9763-c0a45e70eb55.png)